packages/workflow/src/interceptors.ts

```diff
@@ -245,6 +260,64 @@ export interface LocalActivityInput {
   readonly attempt: number;
 }
 
+/**
+ * Input for {@link WorkflowOutboundCallsInterceptor.startNexusOperation}.
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export interface StartNexusOperationInput {
+  input: unknown;
+  endpoint: string;
+  service: string;
+  options: StartNexusOperationOptions;
+  operation: string;
+  seq: number;
+  nexusHeader: Record<string, string>;
+}
+
+/**
+ * Options for starting a Nexus Operation.
+ *
+ * @experimental Nexus support in Temporal SDK is experimental.
+ */
+export interface StartNexusOperationOptions {
+  /**
+   * The end to end timeout for the Nexus Operation.
+   *
+   * Optional: defaults to the maximum allowed by the Temporal server.
+   */
+  scheduleToCloseTimeout?: Duration;
+
+  /**
+   * A fixed, single-line summary for this Nexus Operation that may appear in the UI/CLI.
+   * This can be in single-line Temporal markdown format.
```

Comment:
Am I supposed to know what "Temporal markdown format" is?

Reply:
All the normal Markdown elements except we strip away images, hmtl, and scripts.

Maybe it's worth clarifying that in the docstring